### PR TITLE
remove-should-and-will

### PR DESCRIPTION
**Pull request by kghamilton89**
* remove occurrences of **should** and **will** from `axibase-collector`. This brings this repo to current style standards.

### CODE REVIEW

**Comment by raipc** [comment]
Maybe we'll exclude `*/properties-list.md` from spell checking?

**Comment by rodionos** [comment]
A better option is to not use hash with property types. Instead of headings, use bullet list with backticks. There is no need to create TOC out of property types.

```
## sw.vmw.cluster
```

```
* `sw.vmw.cluster`
```

File: installation.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ tar xvf axibase-collector-v${VERSION}.tar.gz
 
 ## Check Ports
 
-Check that port `9443` is available. If the port is available, output of this command should be blank.
+Check that port `9443` is available. If the port is available, output of this command must be blank.
```
Comment (rodionos) [inline installation.md]: > If the port is available, output of this command must be blank.

better is to take it out.

```
Output of this command is blank if the port is available.
```

File: jobs/examples/activemq/README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ ACTIVEMQ_SUNJMX_START="-Dcom.sun.management.jmxremote \
    -Dcom.sun.management.jmxremote.access.file=${ACTIVEMQ_BASE}/conf/jmx.access"
 ```
 
-This should be the result:
+This must be the result:
```
Comment (rodionos) [inline jobs/examples/activemq/README.md]: Take out "must". Better: `The expected result is` or the `The output is` and similar options.
With should - there are two options. Remove and replace with must to emphasize it's necessary (parameter, setting etc.)

File: jobs/examples/nginx-plus/README.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ The job consists of several settings blocks, each of which is responsible for pr
 
 ![First settings block](./images/ngp_first_settings_block.png)
 
-The result `series` and `property`commands formed by the block should look like this:
+The result `series` and `property`commands formed by the block must look like this:
```
Comment (rodionos) [inline jobs/examples/nginx-plus/README.md]: No space after backticks. Remove "must".

File: jobs/examples/nginx/nginx-configure.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ Add the following lines at the **beginning** of the `location /nginx_status` dir
    deny all;
 ```
 
- For example, if your Collector is located at `10.102.0.6`, the configuration should appear thus:
+ For example, if your Collector is located at `10.102.0.6`, the configuration must appear thus:
```
Comment (rodionos) [inline jobs/examples/nginx/nginx-configure.md]: Remove "must".

File: jobs/examples/pi/export-archive-data.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ This document describes how to export archive data from a [PI Server](http://www
 * PI SQL Data Access Server `1.5+`
 * PI JDBC Driver `1.5+`
 
-PI JDBC Driver should be installed and added as a data source in Collector. Follow the steps in [Axibase Pi Documentation](export-metrics.md#provide-axibase-collector-with-pi-jdbc-driver) to perform this task.
+PI JDBC Driver must be installed and added as a data source in Collector. Follow the steps in [Axibase Pi Documentation](export-metrics.md#provide-axibase-collector-with-pi-jdbc-driver) to perform this task.
```
Comment (rodionos) [inline jobs/examples/pi/export-archive-data.md]: Remove "must". Just say - Install bla-bla... driver.

File: jobs/examples/pi/export-metrics.md [hunk 6]
```diff
@@ -109,6 +109,6 @@ SELECT 1
 ### Verify Metrics in ATSD
 
 * Log in to ATSD.
-* Click on the **Metrics** tab and filter metrics by name, for example find by prefix `ba:`. You should see a list of exported PI tags:
+* Click on the **Metrics** tab and filter metrics by name, for example find by prefix `ba:`. You must see a list of exported PI tags:
```
Comment (rodionos) [inline jobs/examples/pi/export-metrics.md]: Remove "must". Here and in all cases before. Review the content to decide if "must" word is really necessary,

File: installation.md (hunk 1 above)
Comment (kghamilton89) [reply installation.md]: Changed.

File: jobs/examples/activemq/README.md (hunk 2 above)
Comment (kghamilton89) [reply jobs/examples/activemq/README.md]: Changed.

File: jobs/examples/nginx-plus/README.md (hunk 3 above)
Comment (kghamilton89) [reply jobs/examples/nginx-plus/README.md]: Space added, must removed.

File: jobs/examples/nginx/nginx-configure.md (hunk 4 above)
Comment (kghamilton89) [reply jobs/examples/nginx/nginx-configure.md]: Removed.

File: jobs/examples/pi/export-archive-data.md (hunk 5 above)
Comment (kghamilton89) [reply jobs/examples/pi/export-archive-data.md]: Removed.

File: jobs/examples/pi/export-metrics.md (hunk 6 above)
Comment (kghamilton89) [reply jobs/examples/pi/export-metrics.md]: Some of the cases could go either way, but could also be rephrased to remove the need to make the choice, so I just removed all cases.

**Comment by kghamilton89** [comment]
* `./` prefix added to all relative links, anchor check exits w/ `0`
* blacklisted words removed.
* `*/properties-list.md` files updated so headings are now bullets.
* dictionary updated.
* all checks other than `stylecheck` exit with a `0`.
-----
Ostensibly, `stylecheck` should be passing as well but there are [some problems](https://travis-ci.org/axibase/axibase-collector/builds/394492499#L1925) with Travis:

1. Registers the month "May" as the blacklisted word "may."
2. Referencing keywords that do not exist.
3. Referencing blacklisted words that are not present.
4. Referencing backticked keywords, as not backticked.
5. We have a drop-down in the top menu of the Collector UI called **Admin**, so when we reference it by name in the Docs, Travis raises an error. We need to remove this custom rule or change the name of the interface.

**Comment by kghamilton89** [comment]
You'll need to read the raw log to see what I'm talking about, I've reset the build 3 times and it won't print even a quarter of it on the web.

**Comment by kghamilton89** [comment]
There are 2 remaining issues here, both shouldn't be picked up:

* There is an occurrence of 'meteo-stations', name of a metric, which is being flagged as a possessive.
* For some reason `cron` is being flagged for no backticks, but the linter shows the backticked instance in the highlighted example.

**Comment by raipc** [comment]
@annstriganova please take a look

**Pull request merged**